addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js

```diff
@@ -279,3 +279,41 @@ ProductScreen.check.totalAmountIs('1.00');
 PosLoyalty.check.isRewardButtonHighlighted(false);
 
 Tour.register('PosLoyaltyTour10', { test: true, url: '/pos/web' }, getSteps());
+
+startSteps();
+
+ProductScreen.do.clickHomeCategory();
+ProductScreen.do.confirmOpeningPopup();
+
+ProductScreen.do.clickPartnerButton();
+ProductScreen.do.clickCustomer('AAA Partner');
```

Comment:
```suggestion
ProductScreen.do.clickCustomer('AAA Partner');
PosLoyalty.check.customerIs('AAA Partner');
```
wait for the customer to be reflected in the UI before clicking the product.